packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -85,14 +85,14 @@ class DaemonCommand extends FlutterCommand {
       ).run();
       return FlutterCommandResult.success();
     }
-    globals.printStatus('Starting device daemon...');
     final Daemon daemon = Daemon(
       DaemonConnection(
         daemonStreams: DaemonStreams.fromStdio(globals.stdio, logger: globals.logger),
         logger: globals.logger,
       ),
       notifyingLogger: asLogger<NotifyingLogger>(globals.logger),
     );
+    globals.printStatus('Device daemon started');
```

Comment:
Uber nit: I'd prefer for this to end with a period.
```suggestion
    globals.printStatus('Device daemon started.');
```